### ISSUE TITLE
Build Version Deleted IPA

### ISSUE THREAD

**Issue by wes1993** [qnblackcat/How-to-Downgrade-apps-on-AppStore-with-iTunes-and-Charles-Proxy#8]
Hello!!! 
first thanks a lot for you wonderful guide :)
I have one question, how can I find the Build version of a deleted IPA? (I have bought the IPA in my account, I can see in the purchased tab)

The IPA app is Blynk (legacy) not the Blynk IoT

**Comment by qnblackcat** [comment]
Sorry for the late reply, I completely forgot to answer your question. Do you still need it tho?

**Comment by wes1993** [comment]
Yea, I need to Download the Blynk IPA

**Comment by qnblackcat** [comment]
1. Go to your Purchased tab and download the app (Blynk). 
2. Extract the downloaded IPA > Payload > open the `iTunesMetaData.plist` file in any Text editor (ex: Wordpad…)
3. Near the middle of the file, you‘ll see all the build number versions of the app.
4. Now you have the Build version number. Keep trying until you get the version you need. It’ll take a while depending on your luck

**Comment by wes1993** [comment]
Thanks a lot!! Really thanks :)

**Comment by surajreddy** [comment]
Just a note that I had to do this for a paid app, and this comment saved the day. It worked great for downloading an older build of Minecraft. 

For me the `iTunesMetaData.plist` was in the main folder itself (not under Payload).

On windows I followed these steps:
- Download the most current app using iTunes.
- Navigate to the app in explorer.
- Rename the extension to `.zip` from `.ipa`
- Open or extract the app, and open the `iTunesMetaData.plist` file in a text editor.
- I found the build numbers under the key `<key>softwareVersionExternalIdentifiers</key>`
